src/map/entities/battleentity.h

```diff
@@ -234,13 +234,32 @@ enum SLOTTYPE
 // для возможности различить эти орудия при экипировке и избавиться от ошибки
 // использования пуль с арбалетом и арбалетных стрел с огнестрельным оружием (только персонажи)
 
+enum ATTACKTYPE
+{
+    ATTACK_NONE = 0,
+    ATTACK_PHYSICAL = 1,
+    ATTACK_MAGICAL = 2,
+    ATTACK_RANGED = 3,
+    ATTACK_SPECIAL = 4,
+    ATTACK_BREATH = 5,
+};
+
 enum DAMAGETYPE
 {
     DAMAGE_NONE = 0,
     DAMAGE_PIERCING = 1,
     DAMAGE_SLASHING = 2,
     DAMAGE_IMPACT = 3,
-    DAMAGE_HTH = 4
+    DAMAGE_HTH = 4,
+    DAMAGE_ELEMENTAL = 5,
```

Comment:
why would elemental be separate from the elements? how could this ever be set?

Reply:
maybe this was supposed to be NON elemental (such as requiescat)?

Reply:
The idea behind `DAMAGE_ELEMENTAL` is that it can be used as the offset for elements. So you can do `DAMAGE_ELEMENTAL + element` to get the correct damage element. I think right now I only use that in the lua scripts with `dsp.damageTypes` but I was trying to keep parity between the two. We could use it for NON elemental too but I was thinking we could just use `DAMAGE_NONE` for that?

Reply:
i think it's clearer to have a specific non elemental type.  we currently use DAMAGE_NONE to figure out if a character has no weapon equipped, so I wouldn't really want to combine non-elemental into that.  Plus, ELEMENT_NONE would be equivalent to this value anyways, so your use case stays consistent and doesn't need a special case for it

Reply:
I went through and changed all of the non-elemental magic attacks to use `dsp.damageType.ELEMENTAL`. There are still a number of `dsp.damageType.NONE` where I couldn't find any conclusive evidence on the damage type and it was previously set to none.